src/braket/pennylane_plugin/ops.py

```diff
@@ -458,6 +475,16 @@ class YY(Operation):
     par_domain = "R"
     grad_method = "A"
 
+    @staticmethod
+    def decomposition(phi, wires):
+        return [
+            qml.CZ(wires=wires),

```

Comment:
Try CNOT(0, 1) RY(1) CNOT(0, 1)

Reply:
Whoops, that decomposition above is wrong. But it still seems a bit long to me...